src/utils/createTabNavigator.js

```diff
@@ -102,6 +102,11 @@ export default function createTabNavigator(TabView: React.ComponentType<*>) {
       const label = this._getLabelText({ route });
 
       if (typeof label === 'string') {
+        const { index, routesLength } = route;
```

Comment:
This can be replaced with:

```js
const { routes } = this.props.navigation.state;

return `${label} tab, ${routes.indexOf(route) + 1} of ${routes.length}`;
```